### PR TITLE
[ci] merge rayci and ray_release requirements

### PR DESCRIPTION
**Pull request by aslonnie**
so that we have only one single requirements.in file for ci/cd code.

it introduces an additional `colorama` package in the release pipeline, but not a big deal.

### CODE REVIEW

File: release/requirements_buildkite.in [hunk 1]
```diff
@@ -15,3 +15,6 @@ pybuildkite
 PyGithub
 requests
 retry
+
+# Below are requirements only used by ray_ci
+colorama; platform_system == "Windows"
```
Comment (can-anyscale) [inline release/requirements_buildkite.in:20]: colorama is a requirement of anyscale too so maybe just remove this file

Comment (can-anyscale) [reply release/requirements_buildkite.in:20]: also to make it works on windows, can you add `tzdata` as a requirement here

**Review by can-anyscale — APPROVED** [review]

File: release/requirements_buildkite.in (hunk 1 above)
Comment (aslonnie) [reply release/requirements_buildkite.in]: added tzdata and removed colorama.

**Comment by aslonnie** [comment]
not sure if the hermetic rule can work on windows.. seems that on osx and linux they are fine.

**Comment by aslonnie** [comment]
going in~

**Pull request merged**